fluent-syntax/src/parser.js

```diff
@@ -61,21 +60,7 @@ export default class FluentParser {
     while (ps.current()) {
       const entry = this.getEntryOrJunk(ps);
 
-      if (entry === null) {
-        // That happens when we get a 0.4 style section
```

Comment:
I don't think I see where `null` is returned in `getEntryOrJunk`. Did we change it and forget to remove from here?

Reply:
idk :(